packages/reassure-measure/package.json

```diff
@@ -38,8 +38,13 @@
   "dependencies": {
     "mathjs": "^10.1.1"
   },
+  "devDependencies": {
+    "@types/jest": "^28.0.0",
+    "typescript": "^4.5.2"
+  },
   "peerDependencies": {
-    "react": "*"
+    "react": "*",
+    "jest": ""
```

Comment:
it's not a peer dep, one can use vitest

Reply:
Can one use vitest? The README states that jest is required. 🤔